src/components/MARCTable/MARCTableRow.js

```diff
@@ -111,69 +111,49 @@ export const MARCTableRow = ({
 
   const validateTag = useCallback(
     value => {
-      if (actionValue === ADD || actionValue === DELETE || actionValue === MOVE) {
-        return validateMarcTagField(indicator1Value, indicator2Value)(value) || validateRequiredField(value);
-      }
-
-      return null;
+      return validateMarcTagField(indicator1Value, indicator2Value)(value) || validateRequiredField(value);
     },
-    [actionValue, ADD, DELETE, MOVE, indicator1Value, indicator2Value],
+    [indicator1Value, indicator2Value],
   );
   const validateIndicator1 = useCallback(
     value => {
       if (actionValue === ADD) {
         return validateAlphanumericOrAllowedValue(value) || validateMarcIndicatorField(fieldValue, value, indicator2Value);
       }
 
-      if (actionValue === DELETE || actionValue === MOVE) {
-        return validateAlphanumericOrAllowedValue(value, '*') || validateMarcIndicatorField(fieldValue, value, indicator2Value);
-      }
-
-      return null;
+      return validateAlphanumericOrAllowedValue(value, '*') || validateMarcIndicatorField(fieldValue, value, indicator2Value);
     },
-    [ADD, DELETE, MOVE, actionValue, fieldValue, indicator2Value],
+    [actionValue, ADD, fieldValue, indicator2Value],
   );
   const validateIndicator2 = useCallback(
     value => {
       if (actionValue === ADD) {
         return validateAlphanumericOrAllowedValue(value) || validateMarcIndicatorField(fieldValue, indicator1Value, value);
       }
 
-      if (actionValue === DELETE || actionValue === MOVE) {
-        return validateAlphanumericOrAllowedValue(value, '*') || validateMarcIndicatorField(fieldValue, indicator1Value, value);
-      }
-
-      return null;
+      return validateAlphanumericOrAllowedValue(value, '*') || validateMarcIndicatorField(fieldValue, indicator1Value, value);
     },
-    [ADD, DELETE, MOVE, actionValue, fieldValue, indicator1Value],
+    [actionValue, ADD, fieldValue, indicator1Value],
   );
   const validateSubfield = useCallback(
     value => {
       if (actionValue === ADD) {
         return validateAlphanumericOrAllowedValue(value) || validateSubfieldField(fieldValue)(value);
       }
 
-      if (actionValue === DELETE) {
-        return validateRequiredField(value) || validateAlphanumericOrAllowedValue(value, '*') || validateSubfieldField(fieldValue)(value);
-      }
-
-      if (actionValue === MOVE) {
-        return validateAlphanumericOrAllowedValue(value, '*');
-      }
-
-      return null;
+      return validateRequiredField(value) || validateAlphanumericOrAllowedValue(value, '*') || validateSubfieldField(fieldValue)(value);
```

Comment:
Actually I'm not sure that Subfield is required for the Move action.

Reply:
Looks like it is. UIDATIMP-492, Scenario 6: Values other than *, alpha, numeric in Subfield column 6

Reply:
Hmm, at the same time at screenshots there is no value for it.

Reply:
Ok, let's stick to the description.

Reply:
Hmm, probably you are right, i'll leave subfield field as optional for MOVE action